pyrasterframes/src/main/python/docs/nodata-handling.pymd

```diff
@@ -132,29 +135,92 @@ masked.select(rf_no_data_cells('blue_masked'), rf_tile_sum('mask')).show(10)
 
 It's also nice to view a sample.
 
-```python show_masked
+```python
 sample = masked.orderBy(-rf_no_data_cells('blue_masked')).select(rf_tile('blue_masked'), rf_tile('scl')).first()
 display(sample[0])
 ```
 
 And the original SCL data.
 
-```python show_scl
+```python
 display(sample[1])
 ```
 
-## NoData in Arithmetic Operations
+## Changing a Tile's NoData Values
+
+One way to mask a tile is to make a new tile with a user defined NoData value. We will explore this method below. First, lets create a rasterframe from a tile with values of 0, 1, 2, and 3.
+
+```python create_dummy_tile
+tile_size = 100
+x = np.zeros((tile_size, tile_size), dtype='int16')
+
+for i in range(4):
+    x[:, i*tile_size//4:(i+1)*tile_size//4] = i
+x = Tile(x)
+
+rf = spark.createDataFrame(pd.DataFrame(data = 
```

Comment:
Is it necessary to go through Pandas to create a Spark dataframe? Would be nice to avoid teaching the user that ceremony if we can.

Reply:
Maybe Jason knows? From what I could tell using pandas was the cleanest.

Reply:
After talking to Jason, he thinks that going through pandas is the clearest way of doing this. Most online tutorials with pyspark also involve making an intermediate pandas dataframe.

Reply:
I think we could also do something like 

```
spark.createDataFrame([
   Row(tile=x),
])
```

I think also it is a matter of what the user is familiar with.

Reply:
I think I like this approach actually. Changed it.

Reply:
Yeh, that was the approach I prefer.